src/Autocomplete/stories/events.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { withDocs } from 'storybook-readme'
+
+import { GridLayout } from '../../util'
+import { Autocomplete } from '../../index'
+import countries from './countries'
+
+const md = `
+# Events
```

Comment:
😗👌

Reply:
Just keeping it consistent with the other components.

Reply:
Love it.